Add HTTPS support/configuration option for Musicbrainz (#3923)
beetbox/beets: docs/changelog.rst

Other new things:

> * Enable HTTPS support to Musicbrainz by default and add configuration option

Reviewer: ```suggestion
* Enable HTTPS for MusicBrainz by default and add configuration option
```
